gufe/tests/conftest.py

```diff
@@ -46,6 +46,23 @@ def PDBx_181L_path():
         yield str(f)
 
 
+@pytest.fixture
+def offxml_settings_path():
+    with importlib.resources.path('gufe.tests.data', 'offxml_settings.json') as f:
+        yield str(f)
+
+
+@pytest.fixture
+def all_settings_path():
+    with importlib.resources.path('gufe.tests.data', 'all_settings.json') as f:
+        yield str(f)
+
+
+@pytest.fixture
+def dummy_xml_path():
+    with importlib.resources.path('gufe.tests.data', 'note.xml') as f:
+        yield str(f)
```

Comment:
Ooops, this was in here since the openfe forcefield parser throws a different warning if the xml isn't valid, verses the openffxml you passed in isn't valid, so I needed a valid xml file to test that difference in behavior. I then decided it was easier to just catch either error type and forgot to remove this. 

Do we want to handle the error "not valid xml" and "not a valid openffxml" differently?